esp/esp/customforms/views.py

```diff
@@ -25,9 +25,12 @@ def test_func(user):
 
 @user_passes_test(test_func)
 def landing(request):
-    forms = Form.objects.all().order_by('-id')
+    forms = Form.objects.all().order_by('-link_type', '-link_id')
```

Comment:
I think this says to sort by whether program/other, then program id, but I think we should then also sort by form id. Because in the other forms section, it looks very odd to me that the forms aren't very sorted